tests/lvmo/test_lvm_clone_base.py

```diff
@@ -144,23 +81,74 @@ def test_create_clone_from_pvc(
             f"Lvm thin-pool sizePrecent is {lvm.get_lvm_thin_pool_config_size_percent()}"
         )
 
-        logger.info(f"Attaching pod to pvc clone {self.clone.name}")
-        restored_pod_obj = pod_factory(pvc=self.clone, raw_block_pv=self.block)
-        if not self.block:
+        proj_obj = project_factory()
+
+        sc_obj = lvm_storageclass_factory(volume_binding=volume_binding_mode)
+
+        status = constants.STATUS_PENDING
+        if volume_binding_mode == constants.IMMEDIATE_VOLUMEBINDINGMODE:
+            status = constants.STATUS_BOUND
+        pvc_obj = pvc_factory(
+            project=proj_obj,
+            interface=None,
+            storageclass=sc_obj,
+            size=self.pvc_size,
+            status=status,
+            access_mode=self.access_mode,
+            volume_mode=volume_mode,
+        )
+
+        block = False
+        if volume_mode == constants.VOLUME_MODE_BLOCK:
+            block = True
+        pod_obj = pod_factory(pvc=pvc_obj, raw_block_pv=block)
+
+        fs = "fs"
```

Comment:
replace `fs` with `storage_type`

Reply:
Can this be handled in different PR to get this merged ASAP as it block the stable content to be promoted and this PR has a fix for that?  It was here even before @ebenahar

Reply:
yes, sure. I added a few more comments though. 
We can handle them all in a separate PR